index.html

```diff
@@ -0,0 +1,1796 @@
+                              </div>
+                            </li>
+                            <li class="media">
+                              <img class="aj-pic mr-3" src="https://d.newsweek.com/en/full/1587262/wwe-monday-night-raw-aj-styles-returns.jpg?w=95&amp;h=95&amp;l=68&amp;t=25&amp;f=4614454d77ac9ffdd69e32f4f1c575ab" alt="...">
+                              <div class="media-body">
+                                <h6 class="mt-0 mb-1"><a href="/">AJ Styles Returns to WWE 'Monday Night RAW' and Earns MITB Spot</a></h6>
+                              </div>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                
+                <div class="row magazine-section">
+                    <article class="col-lg-7 col-md-12 col-sm-12">
+                        <div class="text-danger font-weight-bold">SUBSCRIBE</div>
+                        <div class="row no-gutters magazine">
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <img src="https://d.newsweek.com/en/full/1585087/may-08-15-2020-cover.jpg?w=170&amp;q=95&amp;f=c4d10af6474d34e586607b280b6589f0" class="card-img" alt="...">
+                            </div>
+                            <div class="col-lg-8 col-md-8 col-sm-12">
+                                <div class="card-body">
+                                    <h5 class="card-title">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h5>
+                                    <p class="card-text text-muted">All Access + Weekly Delivery.Choose A Membership That's Perfect for You!</p>
+                                    <a href="#" class="btn btn-light text-danger">SUBSCRIBE&nbsp;&nbsp;></a>
+                                </div>
+                            </div>  
+                        </div>
+                    </article>
+                    <article class="col-lg-5 col-md-12 col-sm-12">
+                        <div class="text-danger font-weight-bold">NEWSLETTER</div>
+                        <div class="row no-gutters">
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <img src="https://d.newsweek.com/en/full/1541302/starting5-homepage.png?w=397&amp;h=265&amp;q=90&amp;f=742bce9a6f72dd8f19ba2e7fd5db06e5" style="height: 200px;" class="card-img" alt="...">
+                            </div>
+                            <div class="col-lg-8 col-md-8 col-sm-12">
+                                <div class="card-body">
+                                    <h5 class="card-title">The Starting 5</h5>
+                                    <p class="card-text text-muted">See why nearly a quarter of a million subscribers begin their day with the Starting 5.</p>
+                                    <a href="#" class="btn btn-danger">SIGN UP&nbsp;&nbsp;></a>
```

Comment:
Please hide the image for the last section in the phone version.

> Your page
![image](https://user-images.githubusercontent.com/57414789/81482615-ff0cbe00-91fd-11ea-863d-06e9754957d1.png)

> Original 
![image](https://user-images.githubusercontent.com/57414789/81482631-10ee6100-91fe-11ea-95b2-f01cede1e443.png)